.github/components.yml

```diff
@@ -243,6 +243,12 @@ tools:
 docs:
   revalidate: []
   build: []
+  
+docs_snippets:
```

Comment:
we don't have such label in labeler config

Reply:
we have - https://github.com/mryzhov/openvino/blob/gha/smart_ci_snippets/.github/labeler.yml#L63